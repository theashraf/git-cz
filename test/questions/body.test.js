const body = require('../../lib/questions/body');

test('should match snapshot', () => {
  const question = body.createQuestion();

  expect(question).toMatchSnapshot();
});
