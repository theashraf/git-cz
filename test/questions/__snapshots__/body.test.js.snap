// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "message": "Provide a longer description of the change:
 ",
  "name": "body",
  "type": "input",
}
`;
